src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -378,20 +393,24 @@ module.exports = class LevelPlane {
   */
   getPistonState(position) {
     const block = this.getBlockAt(position);
+    let returnValue = "";
 
     if (block.blockType.startsWith("piston") && !block.blockType.startsWith("pistonArm")) {
       block.isPowered = this.powerCheck(position, true);
       if (block.isPowered) {
         this.activatePiston(position);
+        returnValue = "On";
       } else if (!block.isPowered) {
         this.deactivatePiston(position);
+        returnValue = "Off";
```

Comment:
Can you change these "On" -> "Out" and "Off" -> "In" so you don't need the `playPistonSound` switch statement above?